joomla/jissues
# Add php-cs-fixer support

#### Summary of Changes

Adds support for the [PHP-CS-Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) tool as a means to help programmatically review and correct code style issues in PHP code.  This'll cover everything in the `cli` and `src` directories.

#### Testing Instructions

Review the package's documentation to ensure the selected fixers are in line with the project's code style rules and review the changes made to ensure they are consistent with the project's code style rules.

#### What About PHP_CodeSniffer

This is a tool to be used in addition to PHP_CodeSniffer, especially since Joomla's still using PHPCS 1.x which doesn't support auto-fixing.
